src/test/ValidatorKeysTool_test.cpp

```diff
@@ -174,36 +225,62 @@ class ValidatorKeysTool_test : public beast::unit_test::suite
             }
         };
 
+        std::vector <std::string> const noArgs;
+        std::vector <std::string> const oneArg = { "some data" };
+        std::vector <std::string> const twoArgs = { "data", "more data" };
+        std::string const noError = "";
+        std::string const argError = "Syntax error: Wrong number of arguments";
         {
             std::string const command = "unknown";
             std::string const expectedError = "Unknown command: " + command;
-            testArgs (command, keyFile, expectedError);
+            testCommand (command, noArgs, keyFile, expectedError);
+            testCommand (command, oneArg, keyFile, expectedError);
+            testCommand (command, twoArgs, keyFile, expectedError);
         }
         {
             std::string const command = "create_keys";
-            std::string const expectedError = "";
-            testArgs (command, keyFile, expectedError);
+            testCommand (command, noArgs, keyFile, noError);
+            testCommand (command, oneArg, keyFile, argError);
+            testCommand (command, twoArgs, keyFile, argError);
         }
         {
             std::string const command = "create_token";
-            std::string const expectedError = "";
-            testArgs (command, keyFile, expectedError);
+            testCommand (command, noArgs, keyFile, noError);
+            testCommand (command, oneArg, keyFile, argError);
+            testCommand (command, twoArgs, keyFile, argError);
         }
         {
             std::string const command = "revoke_keys";
-            std::string const expectedError = "";
-            testArgs (command, keyFile, expectedError);
+            testCommand (command, noArgs, keyFile, noError);
+            testCommand (command, oneArg, keyFile, argError);
+            testCommand (command, twoArgs, keyFile, argError);
+        }
+        {
+            std::string const command = "sign";
+            testCommand (command, noArgs, keyFile, argError);
+            testCommand (command, oneArg, keyFile, noError);
+            testCommand (command, twoArgs, keyFile, argError);
         }
     }
 
 public:
     void
     run() override
     {
+        // Redirect stdout to tidy test output
+        std::streambuf *old = std::cout.rdbuf();
```

Comment:
It's nice that you tidied up the test output this way.  For extra points you could consider wrapping this in a RAII class similar to what is done here: https://github.com/ripple/rippled/blob/develop/src/test/core/TerminateHandler_test.cpp#L37-L51.  That way in case an exception escapes `run()` then `cout` will still be restored.